TheMovieDatabase/Library/Extensions/Locksmith.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Locksmith.swift
+//  TheMovieDatabase
+//
+//  Created by Evgeny Syrtsov on 4/18/20.
+//  Copyright © 2020 Evgeny Syrtsov. All rights reserved.
+//
+
+import Foundation
+import Locksmith
+
+extension Locksmith {
+    static let loggedUserAccount = "loggedUserAccout"
+    static var sessionId: String? {
+        let dictionary = Locksmith.loadDataForUserAccount(userAccount: Locksmith.loggedUserAccount)
+        guard
+            let sessionId = dictionary?["sessionId"] as? String
+        else {
+            return nil
+        }
+        return sessionId
+    }
+
+    static func save(sessionId: String) throws {
```

Comment:
Okay, you made the method as throws - but it won't throw))))))
'Cause you use do-catch to catch error there so you throw nothing there

You can remove do catch and call just `try Locksmith.updateData(data: ["sessionId": sessionId], forUserAccount: Locksmith.loggedUserAccount)` and it wouldn't be error cause your method throws and if you'll have error inside locksmith it will pass this error to `save(sessionId:)` caller